Sources/OLEKit/DataStream.swift

```diff
@@ -52,7 +52,7 @@ public final class DataReader: Reader {
     precondition(byteOffset + 1 <= data.count)
     defer { byteOffset += 1 }
 
-    return data[byteOffset]
+    return data[data.startIndex + byteOffset]
```

Comment:
Could you clarify please why `data.startIndex` is needed?

Reply:
Because the Data array's index does not start at 0 always, it causes index errors in some cases which I had while developing this PR. So, as my own [DataReader](https://github.com/sboh1214/Hwp-Swift/blob/main/Sources/CoreHwp/Utils/Readers/DataReader.swift), I added startIndex to get correct ranges.

Reply:
Would it make more sense to set `byteOffset` to `data.startIndex` in `init` instead of setting it to `0` like it's done right now?

Reply:
Ah, that approach makes code more simple! I changed in commit [f683820](https://github.com/CoreOffice/OLEKit/pull/9/commits/f683820258a44ed8a51ebcd9fe06986bc67ab55d)